src/components/DatePicker/datepickerPropTypes.js

```diff
@@ -13,13 +13,15 @@ const propTypes = {
      */
     value: PropTypes.oneOfType([PropTypes.instanceOf(Date), PropTypes.string]),
 
+    /* Stores the drafted date/ default value to be set for the user. */
+    defaultValue: PropTypes.oneOfType([PropTypes.instanceOf(Date), PropTypes.string]),
+
```

Comment:
I think we might be missing: `isFormInput`, `inputID` and `shouldSaveDraft` here

Reply:
They are included while importing `fieldPropTypes`.